### DISCUSSION TITLE
How do you align the Action Buttons to the right?

### DISCUSSION THREAD

**Discussion by ahosker** [Power-Components/livewire-powergrid#985] [Q&A] [answered]
I have a few power grid tables on the same page. It looks messy, as the Action Buttons need to be aligned.

I can not see a way in the documentation to align the cell to the right so that the buttons are aligned on the right-hand side.

(p.s. So far -💘 the package, thanks for the hard work)

**Comment by abhaypithadiya** [comment, marked as answer]
Hey!

Publish the Powergrid view file using:

`php artisan vendor:publish --tag=livewire-powergrid-views`

Go to resources\views\vendor\livewire-powergrid\components\actions.blade.php

Here you can change and update column classes.

Hope this helps!
